crates/holochain_types/src/wasmer_types.rs

```diff
@@ -0,0 +1,63 @@
+//! These types manage the consumption and configuration of the `wasmer` library
+//! in terms of middleware (metering), Target, Module, and Store.
+
+use holochain_wasmer_host::prelude::{
+    wasmparser, CompileError, CompilerConfig, CpuFeature, Cranelift, Dylib, Module, Store, Target,
+    Triple,
+};
+use std::str::FromStr;
+use std::sync::Arc;
+use tracing::info;
+use wasmer_middlewares::*;
+
+#[cfg(not(test))]
+/// one hundred giga ops
+pub const WASM_METERING_LIMIT: u64 = 100_000_000_000;
+
+#[cfg(test)]
+/// ten mega ops.
+/// We don't want tests to run forever, and it can take several minutes for 100 giga ops to run.
+pub const WASM_METERING_LIMIT: u64 = 10_000_000;
```

Comment:
Note we probably have to change these to `#[cfg(not(any(test, feature = "test_utils")))]` or something, since `cfg(test)` will not be respected when running tests in downstream crates. But we can fix that later. TODO, cc @abe-njama